networkit/_NetworKit.pyx

```diff
@@ -204,737 +207,7 @@ cdef object toPoint2DVector(const vector[_Point2D]& v):
-		"""
-		return self._this.getAllSimplePaths()
-
-	def forAllSimplePaths(self, object callback):
-		""" More efficient path iterator. Iterates over all the simple paths.
-
-		Parameters
-		----------
-		callback : object
-			Any callable object that takes the parameter path
-		"""
-		cdef PathCallbackWrapper* wrapper
-		try:
-			wrapper = new PathCallbackWrapper(callback)
-			self._this.forAllSimplePaths[PathCallbackWrapper](dereference(wrapper))
-		finally:
-			del wrapper
-
-
-
-cdef extern from "<networkit/distance/APSP.hpp>":
-
-	cdef cppclass _APSP "NetworKit::APSP"(_Algorithm):
-		_APSP(_Graph G) except +
-		vector[vector[edgeweight]] getDistances() except +
-		edgeweight getDistance(node u, node v) except +
-
-cdef class APSP(Algorithm):
-	""" All-Pairs Shortest-Paths algorithm (implemented running Dijkstra's algorithm from each node, or BFS if G is unweighted).
-
-    APSP(G)
-
-    Computes all pairwise shortest-path distances in G.
-
-    Parameters
-	----------
-	G : networkit.Graph
-		The graph.
-    """
-	cdef Graph _G
-
-	def __cinit__(self, Graph G):
-		self._G = G
-		self._this = new _APSP(G._this)
-
-	def __dealloc__(self):
-		self._G = None
-
-	def getDistances(self):
-		""" Returns a vector of vectors of distances between each node pair.
-
- 	 	Returns
- 	 	-------
- 	 	vector of vectors
- 	 		The shortest-path distances from each node to any other node in the graph.
-		"""
-		return (<_APSP*>(self._this)).getDistances()
-
-	def getDistance(self, node u, node v):
-		""" Returns the length of the shortest path from source 'u' to target `v`.
-
-		Parameters
-		----------
-		u : node
-			Source node.
-		v : node
-			Target node.
-
-		Returns
-		-------
-		int or float
-			The distance from 'u' to 'v'.
-		"""
-		return (<_APSP*>(self._this)).getDistance(u, v)
-
-cdef extern from "<networkit/distance/DynAPSP.hpp>":
-
-	cdef cppclass _DynAPSP "NetworKit::DynAPSP"(_APSP):
-		_DynAPSP(_Graph G) except +
-		void update(_GraphEvent ev) except +
-		void updateBatch(vector[_GraphEvent] batch) except +
-
-cdef class DynAPSP(APSP):
-	""" All-Pairs Shortest-Paths algorithm for dynamic graphs.
-
-		DynAPSP(G)
-
-		Computes all pairwise shortest-path distances in G.
-
-		Parameters
-	----------
-	G : networkit.Graph
-		The graph.
-		"""
-	def __init__(self, Graph G):
-		self._G = G
-		self._this = new _DynAPSP(G._this)
-
-	def update(self, ev):
-		""" Updates shortest paths with the edge insertion.
-
-		Parameters
-		----------
-		ev : GraphEvent.
-		"""
-		(<_DynAPSP*>(self._this)).update(_GraphEvent(ev.type, ev.u, ev.v, ev.w))
-
-	def updateBatch(self, batch):
-		""" Updates shortest paths with the batch `batch` of edge insertions.
-
-		Parameters
-		----------
-		batch : list of GraphEvent.
-		"""
-		cdef vector[_GraphEvent] _batch
-		for ev in batch:
-			_batch.push_back(_GraphEvent(ev.type, ev.u, ev.v, ev.w))
-		(<_DynAPSP*>(self._this)).updateBatch(_batch)
-
+# TODO: expose all methods
```

Comment:
?

Reply:
I believe this is the result of a rebase, however it is resolved in a later commit.

Reply:
You could remove the comment using `rebase -i` (doesn't have a high priority but would be nice).

Reply:
Done.